README.md

```diff
@@ -2,25 +2,30 @@
 
 # PROJECT NAME
 
-> Short project decription
+> Web page used to retreive information about Pokemons usin pokeapi v2
```

Comment:
The word retreive is actually spelled as retrieve.
The word using is missing a g
I think it'd be a good idea to capitalize Pokeapi